packages/core/admin/admin/src/content-manager/components/Wysiwyg/Editor.js

```diff
@@ -135,7 +135,8 @@ const EditorStylesContainer = styled.div`
   .CodeMirror {
     /* Set height, width, borders, and global font properties here */
     font-size: ${14 / 16}rem;
-    height: ${({ isExpandMode }) => (isExpandMode ? '100%' : '290px')};
+    height: ${({ isExpandMode }) =>
+      isExpandMode ? '100%' : '412px'}; //  512px(total height) - 48px (header) - 52px(footer)
```

Comment:
I think it needs to be 410px to include also the border, as we do for the block RTE because otherwise it 514px

Reply:
what we do for blocks is that we set the 512px height on the parent, then use flexbox and flex-grow to make sure the input fills the available space. I think it's a better approach, because if we ever add a footer to blocks for example, we won't have to come here and update this formula

Reply:
I tried wrapping editor with `flex` and `flex-grow` for the input, it works. 
But giving fixed height to wrapper, makes input scrollbar hidden, which is actually codemirror styles which is not recommended to change otherwise it will break internal functionality. 

https://github.com/strapi/strapi/pull/18450/files#diff-4f79970ac935faa257775a620493743ed3b07719c348ca31092de6247f0a87faR284

I think its better to keep this way as we are eventually not going to focus on markdown.

Reply:
@madhurisandbhor take a look to these two pictures, they have different height
<img width="1073" alt="block" src="https://github.com/strapi/strapi/assets/2589748/5fe9d717-bc16-4a69-8881-95689824c722">
<img width="1115" alt="markdown" src="https://github.com/strapi/strapi/assets/2589748/4c536452-5eec-4141-bbf7-ebb601e6844a">

Reply:
thank you, updated it.